examples/sync/src/bin/server.rs

```diff
@@ -336,14 +340,16 @@ where
                 if let Err(err) = send_frame(&mut sink, &response_data, MAX_MESSAGE_SIZE).await {
                     info!(client_addr = %client_addr, ?err, "send failed (client likely disconnected)");
                     state.error_counter.inc();
-                    break Ok(());
+                    return Ok(());
                 }
             } else {
                 // Channel closed
-                break Ok(());
+                return Ok(());
             }
         }
     }
+
+    Ok(())
```

Comment:
It seems like we only return `Ok(())`?

Reply:
The loop always returns `()` due to the implicit `break` that is rendered in the macro. We need to add an `Ok(())` after the fact to return after the loop has been exited.